### ISSUE TITLE
Create custom WooCommerce email variables

### ISSUE THREAD

**Issue by clatwell** [Good-Bids/goodbids#364]
Nathan has been working his way through Auction notification email templates. For this ticket, we need to create a bunch of variables that Nathan can pass in to these emails. I'm thinking / hoping that some of our work in the My Account section will extend nicely to create these variables :) 

I'm not actually what we want to call each of these variables, but we don't need to use my suggestions below. @bd-viget can either edit this ticket (or add a comment) with the actual variable names, or @nathan-schmidt-viget can edit this ticket (or add a comment) with the placeholders he used in the emails. 

Quick note on the `optional` fields -- given that some of the Auction fields aren't required, we'll need to be mindful about structuring the emails so we check for the presence of those fields and display the email content conditionally vs. having those fields return `0`. (In other words, if the Auction doesn't have an "Expected High Bid", the "Auction Live for Site Admins" email should omit that row from the table vs. including the row and displaying a $0 value.) 

### Auction 

| Auction Variable    | Variable Name            | Optional? |
|---------------------|--------------------------|-----------|
| Auction Title       | auction.title            |           |
| Auction Page URL    | auction.URL              |           |
| Scheduled Start     | auction.startDate        |           |
| Starting Bid Amount | auction.startingBid      |       |
| Bid Increment       | auction.bidIncrement     |           |
| Auction Goal        | auction.goal             | Yes       |
| Expected High Bid   | auction.expectedHighBid  | Yes       |
| Estimated Value     | auction.estimatedValue | Yes         |
| Scheduled End       | auction.scheduledEndDate |           |
| Bid Extension       | auction.bidExtension     |           |
| Actual End          | auction.actualEndDate    |           |
| Total Bids          | auction.totalBids        |           |
| Total Raised        | auction.totalRaised      |           |
| Winning Bid         | auction.highBid          |           |

### User 

| User Variable    | Variable Name            | Optional? |
|---------------------|--------------------------|-----------|
| User Free Bid Balance      |    user.freeBids        |           |

### User Auction Participation

| User Auction Variable    | Variable Name            | Optional? |
|---------------------|--------------------------|-----------|
| Total Bids      |    auction.user.totalBids        |           |
| Total Donated      |    auction.user.totalDonated        |           |

### Reward Product

| Reward Variable    | Variable Name            | Optional? |
|---------------------|--------------------------|-----------|
| Reward Product Title       | auction.reward.title            |           |
| Reward Checkout URL    | auction.reward.URL              |           |

This represents just our first pass at these variables; there will be more to come as we finalize some additional functionality over the next few weeks, but we'll track those in separate tickets. 

Once this ticket is complete, we'll have a follow-up task for @nathan-schmidt-viget to go back through all the placeholders and update the variables to use the real ones.
